CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Fixed a crash that occurred when `RouteOptions.roadClassesToAvoid` or `RouteOptions.roadClassesToAllow` properties contained multiple road classes.
 * `RoadClasses.tunnel` and `RoadClasses.restricted` are no longer supported in `RouteOptions.roadClassesToAvoid` or `RouteOptions.roadClassesToAllow` properties
 * Added `DirectionsOptions(url:)`, `RouteOptions(url:)` and extended existing `DirectionsOptions(waypoints:profileIdentifier:queryItems:)`, `RouteOptions(waypoints:profileIdentifier:queryItems:)`, `MatchOptions(waypoints:profileIdentifier:queryItems:)` and related convenience init methods for deserializing corresponding options object using appropriate request URL or it's query items. ([#655](https://github.com/mapbox/mapbox-directions-swift/pull/655))
+* `RouteResponse`, `RouteRefreshResponse`, `MatchResponse` and all underlying types now correctly handle unrecogized (foreign) JSON values and preserve them on coding/decoding using `ForeignMemberContainer` and `ForeignMemberClassContainer` implementations. ([#669](https://github.com/mapbox/mapbox-directions-swift/pull/669))
```

Comment:
Typo:

```suggestion
* `RouteResponse`, `RouteRefreshResponse`, `MatchResponse` and all underlying types now correctly handle unrecognized (foreign) JSON values and preserve them on coding/decoding using `ForeignMemberContainer` and `ForeignMemberClassContainer` implementations. ([#669](https://github.com/mapbox/mapbox-directions-swift/pull/669))
```

Should we say something about how this could be useful for using properties in the response that are still in beta?